src/test/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -66,6 +66,33 @@ class BasicWorkflow extends FunSpec with Matchers {
         }
 
         //-----------------------------------------------------------------------------------------------
+        // public symbol search - java.io.File
+
+        val javaSearchSymbol = project.rpcPublicSymbolSearch(List("java", "io", "File"), 2)
+        javaSearchSymbol match {
+          case SymbolSearchResults(List(
+            TypeSearchResult("java.awt.image.ImageObserver", "ImageObserver", 'class,
+              Some(_)),
+            TypeSearchResult("java.io.File", "File", 'class,
+              Some(_)),
+            TypeSearchResult("org.ensime.util.FileUtilSpec", "FileUtilSpec", 'class, None))) =>
```

Comment:
@fommil These indexer results seem wrong - two items:
1. It asks for 2 items and gets 3
2. Search for "java" "io" "File" returns j.a.i.ImageObserver and FileUtilSpec

I accept I may not understand the protocol spec but this feels a bit random.